### ISSUE TITLE
How to adjust learning rate according to the samples_per_gpu and number of GPU?

### ISSUE THREAD

**Issue by MengHao666** [open-mmlab/mmsegmentation#683]
The[ tutorial](https://mmsegmentation.readthedocs.io/en/latest/train.html#train-a-model) said:

`The default learning rate in config files is for 4 GPUs and 2 img/gpu (batch size = 4x2 = 8). Equivalently, you may also use 8 GPUs and 1 imgs/gpu since all models using cross-GPU SyncBN.`

So should learning rate be  linear with` (number of GPUS * samples_per_gpu) ` to achieve similar result as the default configration?

This [issue](https://github.com/open-mmlab/mmsegmentation/issues/380) doesn't give clear answer.

Also [the reference ] that u recomended (https://arxiv.org/pdf/1706.02677.pdf) said `When the minibatch size is multiplied by k, multiply the learning rate by k.`. In this way the learining rate here should only be linear with `samples_per_gpu) '
 
But I have no idea how mmsegmentation process gradient when backward in distributed setting?

Can u provide more information?

**Comment by Junjun2016** [comment]
Hi @MengHao666 
Please check it in MMCV!
